torch/_dynamo/output_graph.py

```diff
@@ -2256,14 +2248,7 @@ def create_graph_input(
                     TracingContext.extract_stack()
                 )
 
-        # unique
-        if name in self.input_name_to_proxy:
-            for i in itertools.count():
-                candidate_name = f"{name}_{i}"
-                if candidate_name not in self.input_name_to_proxy:
-                    name = candidate_name
-                    break
-
+        name = get_unique_name_wrt(name, self.input_name_to_proxy)
```

Comment:
is there a way to get this to always append a number, like before? I'm not sure that it matters but it's nice for the invariant to be "the name is foobar_i" where i is a number

Reply:
Yeah lemme try that, otherwise there's too many tests to update lol.

Reply:
Sad -- either way we'll need to update a bunch of test so I added a `requires_suffix=False` flag to preserve the existing behavior.